CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to ROCm Docs
+
+## Build the docs using VS Code
+
+One can put together a productive environment to author documentation and also test it locally using VS Code with only a handful of extensions. Even though the extension landscape of VS Code is ever changing, here is one example setup that proved useful at the time of writing. In it, one can change/add content, build a new version of the docs using a single VS Code Task (or hotkey), see all errors/warnings emitted by Sphinx in the Problems pane and immediately see the resulting website show up on a locally serving web server.
+
+### Configuring VS Code
+
+1. Install the following extensions:
+
+   - Python (ms-python.python)
+   - Live Server (ritwickdey.LiveServer)
+
+2. Add the following entries in `.vscode/settings.json`
+
+    ```json
+    {
+    	"liveServer.settings.root": "/.vscode/build/html",
+    	"liveServer.settings.wait": 1000,
+    	"python.terminal.activateEnvInCurrentTerminal": true
+    }
+    ```
+
+    The settings in order are set for the following reasons:
+    - Sets the root of the output website for live previews. Must be changed alongside the `tasks.json` command.
+    - Tells live server to wait with the update to give time for Sphinx to regenerate site contents and not refresh before all is don. (Empirical value)
+    - Automatic virtual env activation is a nice touch, should you want to build the site from the integrated terminal.
+
+3. Add the following tasks in `.vscode/tasks.json`
+
+    ```json
+    {
+    	"version": "2.0.0",
+    	"tasks": [
+    		{
+    			"label": "Build Docs",
```

Comment:
Indentation is a bit wonky here, I think, we should do either 2 or 4 spaces. Perhaps 2 on this occasion because this is pretty spammy and overflows when rendered? Your call though.

Reply:
Well, the first 4 spaces is for Markdown to indent the content to the same level as the numbered list (for aesthetics). Then, within the content I used tab to designate indentation and leave it up the the renderer to decide how many characters that is. It's true, that it could be uniform, Markdown would understand the top level spaces be tabs as well.

The thing is, that for Markdown to indent the content, it needs 3-4 spaces in the outermost level (or a tab) to indent. Would one tab then 2 spaces per-level work?

Reply:
Indenting with the list is fine, however, it's the indenting within the codeblock itself that lloked a bit off, see e.g. the opening curly for the `tasks` array.

Reply:
Should look better now.